### PR TITLE
Add docs about manual Functionbeat deployment

### PR DESCRIPTION
**Pull request by dedemorton**
Closes https://github.com/elastic/beats/issues/13372.

### CODE REVIEW

File: x-pack/functionbeat/docs/deploying.asciidoc [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+`ENABLED_FUNCTIONS=function-name-1,function-name-2`:: Specifies a comma-
+separated list of functions that are enabled in the configuration file. For
+example, to package functions called `my-kinesis` and `my-cloudwatch-logs`, run:
++
+*linux and mac*:
++
+[source, shell]
+----
+export BEAT_STRICT_PERMS=false
+export ENABLED_FUNCTIONS=my-kinesis,my-cloudwatch-logs
```
Comment (ph) [inline x-pack/functionbeat/docs/deploying.asciidoc]: I would recommend a lambda per function, they are just easier to manage that way.

File: x-pack/functionbeat/docs/deploying.asciidoc [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+same path as the configured +{beatname_lc}.yml+ file. 
+
+. Deploy the package, using the infrastructure and automation supported by your
+cloud provider. For example, to deploy the package to AWS,
+<<export-cloudformation-template,export the AWS CloudFormation template>>
+managed by {beatname_uc}, and modify it to work with your infrastructure. 
+
+// REVIEWERS: I could show the aws cloudformation deploy
+// command here, but I think it might be better to avoid doing so. Users should
+// already know this, no?
```
Comment (ph) [inline x-pack/functionbeat/docs/deploying.asciidoc]: I agree to that.

**Comment by dedemorton** [comment]
@ph I made the change you suggested. Can you check it?

File: x-pack/functionbeat/docs/overview.asciidoc [hunk 3]
```diff
@@ -62,6 +56,7 @@ data.
 
 image::./images/functionbeat-pipeline-cloudwatchlogs.png["{beatname_uc} collects events generated by CloudWatch logs"]
 
+//REVIEWERS: Should we uncomment this section now?
```
Comment (dedemorton) [inline x-pack/functionbeat/docs/overview.asciidoc]:
```suggestion
```

**Comment by dedemorton** [comment]
jenkins test this again

**Pull request merged**